ScreenManager.lua

```diff
@@ -533,6 +528,7 @@ end
 --
 function ScreenManager.registerCallbacks( callbacks )
     local registry = {};
+    local function null() end
```

Comment:
They are... I'm sorry hahaha. Should I put a `return;` inside there just like you did in `Screen.lua`?